src/phoenix/server/api/types/ExperimentRunAnnotation.py

```diff
@@ -17,17 +17,25 @@
 class ExperimentRunAnnotation(Node):
     id: NodeID[int]
     db_record: strawberry.Private[Optional[models.ExperimentRunAnnotation]] = None
+    _raw_annotation: strawberry.Private[Optional[Dict[str, Any]]] = None
```

Comment:
```suggestion
    _raw_annotation: strawberry.Private[Optional[Dict[str, Any]]] = None
```

Suggestion: I think it's worth defining a dataclass or typed dict for the raw annotation type. Would mean we don't need to worry about using `get` with fallbacks.

Reply:
why not just use db_record with id = -1? it seems to me that's equivalent without all the extra code